utils/logger.go

```diff
@@ -0,0 +1,122 @@
+package utils
+
+import (
+	"strings"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// Logger is the wrapper of zap.SugaredLogger.
+// By default, the logger writes the output
+// to a JSON-format file.
+type Logger struct {
+	zLogger *zap.SugaredLogger
+}
+
+// A LoggerConfig contains the running environment
+// which is either "development" or "production",
+// and the path of file to write the logging output to.
+// Logging is enabled at DebugLevel and above in development mode,
+// at InfoLevel and above in production mode.
+type LoggerConfig struct {
+	Environment string `toml:"env"`
+	Path        string `toml:"path"`
+}
+
+func NewLogger(conf *LoggerConfig) *Logger {
+	zLevel := zap.NewAtomicLevel()
+	switch {
+	case strings.EqualFold("development", conf.Environment):
+		zLevel.SetLevel(zap.DebugLevel)
+	case strings.EqualFold("production", conf.Environment):
+		zLevel.SetLevel(zap.InfoLevel)
+	default:
+		panic("Environment must be either development or production")
+	}
+
+	zConfig := &zap.Config{
+		Level:             zLevel,
+		Development:       strings.EqualFold("development", conf.Environment),
+		Encoding:          "json",
+		DisableStacktrace: true, // turn off stacktrace capturing
```

Comment:
I'm wondering if it might be useful to enable this feature optionally in dev mode? Maybe pass in a flag that will set this to `false` if the developer chooses?